dockstore-webservice/src/main/resources/openapi.yaml

```diff
@@ -825,6 +837,12 @@ paths:
           required: false
           schema:
             type: string
+        - name: checker
+          in: query
+          description: Return only checker workflows
+          required: false
```

Comment:
set default false

Reply:
(this is generated)

Reply:
ya, i was thinking to set it as false in the `@ApiParam(..., defaultValue=false)`

Reply:
I think the API declaration is generated too (TRS swagger -> server API)